.travis.yml

```diff
@@ -27,10 +27,15 @@ install: mvn install -q -DskipTests=true
 
 script: mvn test -q
 
+# Bucket-type to activate before running Riak KV tests.
```

Comment:
Actually, the only way to create a bucket-type is via riak-admin commands, so it's not possible. However, I think I could modify the test class to execute those commands from within there, but I would at least keep the `sudo: required` line as it is. This is because the aforementioned commands need to be executed with `sudo`.